### DISCUSSION TITLE
SFTPToGCSOperator except file and folder

### DISCUSSION THREAD

**Discussion by superbeer** [apache/airflow#28577] [Ideas]
### Description

SFTPToGCSOperator except file and folder 

### Use case/motivation

i'm use * for sftp  but can not except  folder not want to gcs
├── 1.txt
├── 2.txt
└── exception
    └── 3.txt
ignore folder exception

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by potiuk** [comment]
Assigned you!
